### PR TITLE
logging

### LINKED ISSUE #104: logging
It is necessary to clear the warnings in order to find the error that caused the PRs to fail. It is necessary to add simple logging to track the output of tests.

### PR DESCRIPTION
**Pull request by advasileva**
Clear warning, add logging. Closes #104

### CODE REVIEW

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@advasileva, in my opinion, the current logging system doesn't provide any useful information. When Maven calls tests, it already prints information about the started tests, so adding duplicate log entries is unnecessary.

Instead, I suggest placing the log statements somewhere in the middle of parsing and transformation. Also, it's a good practice to add context to the logs to make them more informative. You might find [this article](https://www.yegor256.com/2015/12/01/rethrow-exceptions.html) about re-throwing exceptions helpful in implementing context in your logs.

**Comment by advasileva** [comment]
@volodya-lombrozo At the moment I need logs for tests, not parsing and transformations. This is due to the fact that compilation tests take quite a long time and I need to make sure that these are the reasons for the PR merging timeout.

I agree that it is better to move logging closer to the intended problem, so I only left logs in `dataize`. It seems to me that there is not much to expand the context here, because it is just a method called in `assert`.

**Comment by advasileva** [comment]
@volodya-lombrozo What do you think?

File: src/test/java/org/eolang/speco/SpecoEoTest.java [hunk 1]
```diff
@@ -139,6 +140,7 @@ private static String[] dataize(final String target) throws IOException, Interru
             executor = "bash";
             flag = "-c";
         }
+        Logger.debug(this, String.format("Started compilation for %s", target));
```
Comment (volodya-lombrozo) [inline src/test/java/org/eolang/speco/SpecoEoTest.java:143]: @advasileva In that case in the log you will see:
```
DEBUG org.eolang.speco.SpecoEoTest MDC= - Started compilation for /tmp/junit3607004110884547520/output
```
It doesn't tell something about the test case and about the code which will be transformed. Maybe we can add more context here? What do you think?

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]

File: src/test/java/org/eolang/speco/SpecoEoTest.java (hunk 1 above)
Comment (advasileva) [reply src/test/java/org/eolang/speco/SpecoEoTest.java]: @volodya-lombrozo I added the beginning of the test to the logs to make it clearer which test is currently being executed. In this test, I have only the text of the input program (even without the name of the test), so I didn't come up with anything better. I can also add a description of the test to `.yaml` test and outputs it as a context

Comment (advasileva) [reply src/test/java/org/eolang/speco/SpecoEoTest.java]: @volodya-lombrozo Done, created an issue #105.

**Comment by advasileva** [comment]
@volodya-lombrozo Thanks for review.
@yegor256 I agree that this PR can be merged.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**